content/pricing/free.textile

```diff
@@ -55,6 +55,12 @@ To downgrade your account to a Free package:
 2. Log in to your "account":https://ably.com/login and select *Billing* from the *Account* menu.
 3. Click the *Downgrade* button.
 
-If you downgrade to a Free package, you will finish the month on your current package with its associated limits and benefits. At the beginning of the following month you will receive a final invoice covering your last month's consumption and package fee. Once the final invoice has been paid, your payment details are removed from the system.
```

Comment:
I don't think this is correct - I think the final package fee will still be charged.